controller/src/controller/main_systems/system_monitor.py

```diff
@@ -401,7 +428,8 @@ async def _handle_comm_from_cloud_comm(self) -> None:
                 case invalid_comm:
                     raise NotImplementedError(f"Invalid communication from CloudComm: {invalid_comm}")
 
-            if system_state_updates:
+            # only update system status if not in offline state, affects starting up in offline
+            if system_state_updates and system_state["system_status"] != SystemStatuses.OFFLINE_STATE:
```

Comment:
We might want to skip or delay the cloud comm stuff instead if we're in offline mode